src/workloads/scale/BulkLoading.yml

```diff
@@ -0,0 +1,287 @@
+      dataField5: { ^FastRandomString: { length: 50 } },
+    }
+
+  # This workload uses a batch size of 100 for the bulk writes.
+  # We evaluated the impact of different batch sizes and the document throughput
+  # does not appear to be impacted by the batch size.
+  - &WriteOp { WriteCommand: insertOne, Document: *Document }
+  - &WriteOps10x [
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+      *WriteOp, *WriteOp, *WriteOp, *WriteOp, *WriteOp,
+    ]
+  - &WriteOps100x {
+      ^FlattenOnce:
+        [
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+          *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
+        ],
+    }
+  - &UpsertOp {
+      WriteCommand: updateOne,
+      Filter: { sensorId: { ^RandomInt: { min: 0, max: 200000000 } } },
+      Update:
+        {
+          $set:
+            { eventId: 0, dataField1: { ^FastRandomString: { length: 10 } } },
+        },
+      OperationOptions: { Upsert: true },
+    }
+  - &UpsertOps10x [
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+      *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp, *UpsertOp,
+    ]
+  - &UpsertOps100x {
+      ^FlattenOnce:
+        [
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+          *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x, *UpsertOps10x,
+        ],
+    }
+
+ActorTemplates:
+  - TemplateName: BulkWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: "unset" } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: bulkWrite
+                OperationCommand:
+                  WriteOperations:
+                    { ^Parameter: { Name: "Operations", Default: "unset" } }
+  - TemplateName: NormalWriterTemplate
+    Config:
+      Name: { ^Parameter: { Name: "Name", Default: "unset" } }
+      Type: CrudActor
+      Threads: { ^Parameter: { Name: "Threads", Default: "unset" } }
+      Phases:
+        OnlyActiveInPhases:
+          Active:
+            { ^Parameter: { Name: "OnlyActiveInPhase", Default: "unset" } }
+          NopInPhasesUpTo: *NumPhases
+          PhaseConfig:
+            Duration: { ^Parameter: { Name: "Duration", Default: "unset" } }
+            Database: *DbName
+            Collection: *CollectionName
+            Operations:
+              - OperationName: insertOne
+                OperationCommand:
+                  Document: *Document
+
+Actors:
+  # This loads about 35 GB of data to warm up the DB
```

Comment:
nit: Maybe '_Each invocation of this actor bulk loads about 35 GB of data to populate the DB_'?